### DISCUSSION TITLE
column roles.default_time_entry_activity_id does not exist

### DISCUSSION THREAD

**Discussion by 17635386421** [docker-library/redmine#321] [General]
update postgres image，reimport data，repo error ‘’column roles.default_time_entry_activity_id does not exist‘’ while open issue page
![image](https://github.com/docker-library/redmine/assets/55479115/de804ac6-f560-4128-883c-b432585ff5ad)
![image](https://github.com/docker-library/redmine/assets/55479115/206eb981-155a-4ee9-a8da-79f5d6e091a2)

**Comment by mer0m** [comment]
Hi @17635386421 ,

I just had the same issue in a migration.

In prod, I solved it with:

```bash
cd /path/to/redmine/
bundle exec rake db:migrate RAILS_ENV=production
bundle exec rake redmine:plugins:migrate RAILS_ENV=production
bundle exec rake tmp:cache:clear RAILS_ENV=production
```

See https://www.redmine.org/issues/40181
